ui/component/fileRenderInitiator/index.js

```diff
@@ -32,23 +34,25 @@ const select = (state, props) => {
   const { claim_id: channelClaimId } = channelClaim || {};
 
   return {
-    claimId,
+    authenticated: selectUserVerifiedEmail(state),
+    autoplay: selectClientSetting(state, SETTINGS.AUTOPLAY_MEDIA),
     channelClaimId,
+    claimId,
+    claimIsMine: selectClaimIsMine(state, claim),
     claimThumbnail: getThumbnailFromClaim(claim),
+    claimWasPurchased: selectClaimWasPurchasedForUri(state, uri),
+    costInfo: selectCostInfoForUri(state, uri),
     fileInfo: makeSelectFileInfoForUri(uri)(state),
-    obscurePreview: selectShouldObscurePreviewForUri(state, uri),
-    isPlaying: selectFileIsPlayingOnPage(state, uri),
     insufficientCredits: selectInsufficientCreditsForUri(state, uri),
-    autoplay: selectClientSetting(state, SETTINGS.AUTOPLAY_MEDIA),
-    costInfo: selectCostInfoForUri(state, uri),
-    renderMode: makeSelectFileRenderModeForUri(uri)(state),
-    claimWasPurchased: selectClaimWasPurchasedForUri(state, uri),
-    authenticated: selectUserVerifiedEmail(state),
     isCurrentClaimLive: selectIsActiveLivestreamForUri(state, uri),
     isLivestreamClaim: isStreamPlaceholderClaim(claim),
-    purchaseContentTag: selectPurchaseTagForUri(state, props.uri),
+    isPlaying: selectFileIsPlayingOnPage(state, uri),
+    obscurePreview: selectShouldObscurePreviewForUri(state, uri),
+    purchaseContentTag: selectPurchaseTagForUri(state, uri),
     purchaseMadeForClaimId: selectPurchaseMadeForClaimId(state, claimId),
-    claimIsMine: selectClaimIsMine(state, claim),
+    renderMode: makeSelectFileRenderModeForUri(uri)(state),
+    rentalTag: selectRentalTagForUri(state, uri),
+    validRentalPurchase: Boolean(selectValidRentalPurchaseForClaimId(state, claim.claim_id)),
```

Comment:
:small_orange_diamond:  I don't have an example that would cause `claim` to be unresolved (deleted claim in an embed, perhaps), but given that other selectors above handled that, we should just replace `claim.claim_id` with the dereferenced `claimId` just to be safe (avoid "accesing claim_id of null")